### PR TITLE
timeout for curl

### PR DESCRIPTION
**Pull request by jhammock**
I took a stab, not sure if it's grammatical...

### CODE REVIEW

File: archives.groovy [hunk 1]
```diff
@@ -87,7 +87,7 @@ def updateMonitors() {
 }
 
 def requestUpdate() {
-  sparkRequest = '''curl --verbose -X POST http://@@HOST@@:7077/v1/submissions/create --header "Content-Type:application/json;charset=UTF-8" --data '{
+  sparkRequest = '''curl --verbose --max-time <10000> -X POST http://@@HOST@@:7077/v1/submissions/create --header "Content-Type:application/json;charset=UTF-8" --data '{
```
Comment (jhpoelen) [inline archives.groovy]: suggest to remove the "<" and ">" . The submit / status update commands should be pretty short lived, so, a time-out of 30 seconds should be sufficient. You should be able to test in your mac terminal.

**Comment by jhpoelen** [comment]
One way to test it is to ... merge in the changes and use it!

**Comment by jhammock** [comment]
OK, iNat dataset updated, updateMonitors is running...

**Comment by jhpoelen** [comment]
Great! Looks like your fix is working http://archive.guoda.bio/job/update%20monitors/128/console - 
```
curl --silent --max-time 60 http://mesos07.acis.ufl.edu:7077/v1/submissions/status/driver-20190418131208-11978873
[Pipeline] echo
status: [{
  "action" : "SubmissionStatusResponse",
  "driverState" : "RUNNING",
  "message" : "task_id {\n  value: \"driver-20190418131208-11978873\"\n}\nstate: TASK_RUNNING\nslave_id {\n  value: \"dabbc59d-84c3-4fb3-8baa-045c55e020fb-S4\"\n}\ntimestamp: 1.5556075348839371E9\nexecutor_id {\n  value: \"driver-20190418131208-11978873\"\n}\nsource: SOURCE_EXECUTOR\nuuid: \"\\357\\266XlaH\\345\\210\\373\\206\\261\\262\\244\\016\\333\"\ncontainer_status {\n  network_infos {\n    ip_addresses {\n      ip_address: \"10.13.44.39\"\n    }\n  }\n  executor_pid: 26832\n}\n",
  "serverSparkVersion" : "2.2.0",
  "submissionId" : "driver-20190418131208-11978873",
  "success" : true
}]
```

**Pull request merged**